src/footer/Footer/Footer.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, chakra, useMultiStyleConfig } from "@chakra-ui/react";
+import { Image, List, Link, Icon, IconNames, Logo } from "@nypl/design-system-react-components";
+import { forwardRef } from "react";
+
+import linkData from "./footerLinks";
+
+interface FooterProps {
+  /** Additional class name to add. */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+}
+
+/**
+ * This `Footer` component renders the NYPL-branded footer elements such
+ * as navigational NYPL.org links, social media links, copyright, and
+ * NYPL building facade image.
+ */
+export const Footer = chakra(
+  forwardRef<HTMLDivElement, FooterProps>(
+    ({ className, id = "footer", ...rest }, ref?) => {
+      const styles = useMultiStyleConfig("Footer", {});
```

Comment:
AH! I was looking for the styles and realized that it might be pulling in the Footer and Header styles from the DS itself. This is not a problem right now but we should update it later on. _If_ the Footer and Header are completely removed from the DS, this won't work.

I'm not sure the best way to update this right now, but it's not crucial for this PR. So I'll create a new ticket.

I think we may need to create a new "theme" that contains the header and footer styles and pass it to the DSProvider. That may be the last step needed to remove the Header/Footer dependencies from the DS.

Reply:
Sounds good!